app/controllers/csv_preview_controller.rb

```diff
@@ -73,4 +81,8 @@ def redirect_to_placeholder
     expires_in(1.minute, public: true)
     redirect_to placeholder_path
   end
+
+  def hostname_includes_draft_assets?
+    request.hostname.split(".").first == "draft-assets"
```

Comment:
Minor thing – I wonder if this is any more readable?

```suggestion
    request.hostname.start_with? "draft-assets"
```